devel/git-town/Portfile

```
@@ -3,8 +3,8 @@
 PortSystem          1.0
 PortGroup           golang 1.0
 
-go.setup            github.com/git-town/git-town 18.3.2 v
-go.package          github.com/git-town/git-town/v18
+go.setup            github.com/git-town/git-town 22.2.0 v
+go.package          github.com/git-town/git-town/v22
```

We can probably use some Tcl here so that this doesn't have to be manually changed on each major version update:

```suggestion
set major_ver       [lindex [split ${github.version} .] 0]
go.package          github.com/git-town/git-town/v${major_ver}
```